lib/get-config.js

```diff
@@ -283,6 +286,51 @@ function processIgnores(config) {
   config.ignore = config.ignore.map((pattern) => micromatch.matcher(pattern));
 }
 
+/**
+ * we were passed a rule, add the rule being passed in, to the config.
+ * ex:
+ * rule:severity
+ * no-implicit-this:["error", { "allow": ["some-helper"] }]
+ */
+function getRuleFromString(rule) {
+  const indexOfSeparator = rule.indexOf(':') + 1;
+
+  // we have to split based on the index of the first instance of the separator because the separator could exist in the second half of the rule
+  const name = rule.substring(0, indexOfSeparator - 1);
+  let severity = rule.substring(indexOfSeparator);
```

Comment:
FWIW, I think _technically_ this is not the "severity", it is the config for the rule (which **might** just be `"error"` (the severity) but could be any valid config value e.g. `["error", whateverGoesHere]`).

Reply:
What we rather call this?

Reply:
AFAICT it is the `ruleConfig`, what do you think?

Reply:
makes sense to me, renamed that in the most recent diff.